vxlan/vxlan.go

```diff
@@ -0,0 +1,273 @@
+// Package vxlan provides the capabilities to create a VXLAN overlay network
+package vxlan
+
+import (
+	"net"
+	"sync"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/pkg/errors"
+	"github.com/rancher/go-rancher-metadata/metadata"
+	"github.com/vishvananda/netlink"
+)
+
+const (
+	changeCheckInterval = 5
+	metadataURL         = "http://rancher-metadata/2016-07-29"
+	vxlanMACRange       = "0E:00:00:00:00:00"
+	vxlanPort           = 4789
+)
+
+var (
+	BridgeName               = "docker0"
```

Comment:
Could you please change these variables to be just Default values?
And create the variables inside the vxlan struct and assign values to those, instead of using these?

DefaultBridgeName
DefaultVxlanMTU
...